remo/reports/tests/test_api.py

```diff
@@ -69,3 +73,104 @@ def test_get_remo_url(self):
         data = ActivitiesDetailedSerializer(
             report, context={'request': request}).data
         ok_(report.get_absolute_url() in data['remo_url'])
+
+
+class TestActivitiesKPIView(RemoTestCase):
```

Comment:
This is part of DRF functionality and it should be tested in that level. IMHO there is no need to test the functionality provided by the framework.